.github/workflows/buildtools.yml

```diff
@@ -25,15 +25,19 @@ jobs:
         run: |
           btrev=""
           pageNum=1
+          releaseUrl="${{ github.repository }}/releases/tag/buildtools-${{ env.HAIKU_ARCH }}-"
           while [ "$btrev" == "" ];
           do
+            # Fetch using GitHub API
             json=$(curl -s ${{ github.api_url }}/repos/${{ github.repository }}/releases?page=$pageNum)
             pageNum=$((pageNum + 1))
-            echo $json | jq -e -r '.[] | .html_url'
-            if [ "$?" != "0" ]; then
+            if [ $(echo $json | jq length) -eq 0 ]; then
+              # This means that we've passed the end and reached an empty array
               break
             fi
-            btrev=$(echo $json | jq -r '.[] | .html_url | select(contains("buildtools"))' | sed -n 's/^.*releases\/tag\/buildtools-${{ env.HAIKU_ARCH }}-\([^"]*\).*$/\1/p' | head -n 1)
+            revisions=$(echo $json | jq -e -r ".[] | .html_url | select(contains(\"buildtools\"))[${#releaseUrl}:]") \
```

Comment:
For some strange reasons storing all of this into a variable would convert all newlines into spaces, breaking the `head` command below that relies on newlines.

Reply:
Ah, so it does. Changed to an array.